AndroidAnnotations/androidannotations-core/androidannotations/src/main/java/org/androidannotations/internal/core/handler/RootContextHandler.java

```diff
@@ -82,13 +82,13 @@ public void assignValue(JBlock targetBlock, IJAssignmentTarget fieldRef, EBeanHo
 		} else {
 			AbstractJClass extendingContextClass = getEnvironment().getJClass(typeQualifiedName);
 
-			JConditional cond = getInvocationBlock(holder)._if(holder.getContextRef()._instanceof(extendingContextClass));
```

Comment:
could you please rebase on develop and check if the header `Copyright (C) 2016-2019 the AndroidAnnotations project` has been added to the file? if not please add it.

Reply:
@dodgex done for this PR